### PR TITLE
CT-1345: OpenTok Network Test

### PR DESCRIPTION
**Pull request by tysonxor**
WIP! Dont merge please

### CODE REVIEW

**Comment by tysonxor** [comment]
accidental label - this is not a bug ;-)

**Comment by k7k0** [comment]
After my lastest merge, remember to add this new files to the
Modules/Opentok folder reference inside xcode (drag and drop)

File: OTNetworkTest.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    
+}
+- (void)runConnectivityTestWithApiKey:(NSString*)apiKey
+                            sessionId:(NSString*)sesssionId
+                                token:(NSString*)token
+                   executeQualityTest:(BOOL)needsQualityTest
+                  qualityTestDuration:(int)qualityTestDuration
+                              resolve:(RCTPromiseResolveBlock)resolve
+                               reject:(RCTPromiseRejectBlock)reject
+                             delegate:(id<OTNetworkTestDelegate>)delegate;
```
Comment (ghost) [inline OTNetworkTest.h:30]: @adamtysonsmith `resolve` and `reject` need to be last args for this to work I believe https://facebook.github.io/react-native/docs/native-modules-ios.html#promises

Comment (ghost) [reply OTNetworkTest.h:30]: Nvm, this is not the exported method. This can be ignored.

Comment (tysonxor) [reply OTNetworkTest.h:30]: Ripped those out!

**Comment by tysonxor** [comment]
RCTOpenTokNetworkTest is really the only objc class I worked on, the rest is network test code from OpenTok that has not been modified

**Review by k7k0 — APPROVED** [review]

**Comment by k7k0** [comment]
Merged, remember to PR the project changes in our Mobile project!

**Pull request merged**